packages/difft.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
+require_relative 'difftastic'
```

Comment:
Do we really need an alias for this? After all, it's named upstream as difftastic.

Reply:
It would be nice to have some alias functionality in crew, like we have our table of adjustments in `fixup.rb` such that crew knows when there is an alias without having to create a separate package file.
maybe an aliases line in the package?

Reply:
Also, `diift` is the name of the binary, so people might look for a `difft` package.

Reply:
> Also, `difft` is the name of the binary, so people might look for a `difft` package.

User can use `crew whatprovides` to search the package with the command name

Reply:
Most users don't know how to use `crew whatprovides`.  My policy is to always provide a post install message if the package name differs from the cli command.  It helps avoid confusion.